eth-contracts/migrations/2_token_migration.js

```diff
@@ -1,5 +1,9 @@
 const AudiusToken = artifacts.require('AudiusToken')
 
 module.exports = (deployer, network, accounts) => {
-    deployer.deploy(AudiusToken)
+    deployer.then(async () => {
+        const token = await deployer.deploy(AudiusToken, { from: accounts[0] })
+        // note - for some reason this call fails if { from } is passed
```

Comment:
should this comment be updated or the from removed? p sure its accounts[0] by default anyway